Content/1.1.4 - Interaction Review/interactionReview.json

```diff
@@ -0,0 +1,34 @@
+{
+	"interactionReview_possibleDappCalls": "Possible dApp calls",
+	"interactionReview_customizeGuaranteesButtonTitle": "Customize Guarantees",
```

Comment:
from line 3 till the end, there is basically every string that used to be under `transactionReview_` that is now part of a common view, and therefore placed here. In this process, a lot of unused strings have been removed as well.

For further reference, these are the Strings that must be kept in `transactionReview.json`, since they don't correspond to shared components:
```
transactionReview_xrdAmount
transactionReview_feePayerValidation_insufficientBalance
transactionReviewfeePayerValidation_feePayerRequired
transactionReview_feePayerValidation_linksNewAccount
transactionReview__networkFee_heading
transactionReview__networkFee_customizeButtonTitle
transactionReview__guarantees_title
transactionReview__guarantees_subtitle
transactionReview__guarantees_applyButtonText
transactionReview__guarantees_setGuaranteedMinimum
transactionReview__submitTransaction_txID
transactionReview__nonConformingManifestWarning_title
transactionReview__nonConformingManifestWarning_message
```